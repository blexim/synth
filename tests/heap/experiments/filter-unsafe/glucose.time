WARNING! DIMACS header mismatch: wrong number of variables.
Command exited with non-zero status 10
0.83user 0.02system 0:00.86elapsed 99%CPU (0avgtext+0avgdata 45568maxresident)k
0inputs+16outputs (0major+15445minor)pagefaults 0swaps
