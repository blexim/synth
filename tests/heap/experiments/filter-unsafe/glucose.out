c
c This is glucose 4.0 --  based on MiniSAT (Many thanks to MiniSAT team)
c
c ========================================[ Problem Statistics ]===========================================
c |                                                                                                       |
c |  Number of variables:         59603                                                                   |
c |  Number of clauses:          249755                                                                   |
c |  Parse time:                   0.07 s                                                                 |
c |                                                                                                       |
c | Preprocesing is fully done
c |  Eliminated clauses:           2.51 Mb                                                                |
c |  Simplification time:          0.33 s                                                                 |
c |                                                                                                       |
c ========================================[ MAGIC CONSTANTS ]==============================================
c | Constants are supposed to work well together :-)                                                      |
c | however, if you find better choices, please let us known...                                           |
c |-------------------------------------------------------------------------------------------------------|
c |                                |                                |                                     |
c | - Restarts:                    | - Reduce Clause DB:            | - Minimize Asserting:               |
c |   * LBD Queue    :     50      |   * First     :   2000         |    * size <  30                     |
c |   * Trail  Queue :   5000      |   * Inc       :    300         |    * lbd  <   6                     |
c |   * K            :   0.80      |   * Special   :   1000         |                                     |
c |   * R            :   1.40      |   * Protected :  (lbd)< 30     |                                     |
c |                                |                                |                                     |
c ==================================[ Search Statistics (every  10000 conflicts) ]=========================
c |                                                                                                       |
c |          RESTARTS           |          ORIGINAL         |              LEARNT              | Progress |
c |       NB   Blocked  Avg Cfc |    Vars  Clauses Literals |   Red   Learnts    LBD2  Removed |          |
c =========================================================================================================
c |       46         0      217 |    8218    68215   284315 |     2     6132      501     3704 | 12.352 % |
c last restart ## conflicts  :  223 281 
c =========================================================================================================
c restarts              : 57 (193 conflicts in avg)
c blocked restarts      : 1 (multiple: 1) 
c last block at restart : 57
c nb ReduceDB           : 2
c nb removed Clauses    : 3704
c nb learnts DL2        : 538
c nb learnts size 2     : 159
c nb learnts size 1     : 13
c conflicts             : 11028          (13227 /sec)
c decisions             : 29390          (0.00 % random) (35251 /sec)
c propagations          : 3290882        (3947180 /sec)
c conflict literals     : 134582         (23.09 % deleted)
c nb reduced Clauses    : 96
c CPU time              : 0.83373 s

s SATISFIABLE
